Source/FullScreenPokemon.ts

```diff
@@ -8635,7 +8635,15 @@ module FullScreenPokemon {
         */
 
         /**
+         * Sets the game state to a new Map, resetting all Things and inputs in the
+         * process. The mod events are fired.
          * 
+         * @param name   The name of the Map.
+         * @param location   The name of the location within the Map (by
+         *                   default, 0 for the first in Array form).
+         * @param noEntrance    Whether or not an entry function is used to
```

Comment:
Should be more accurate. `noEntrance` is whether the entry function should be *skipped* (by default, false).